### ISSUE TITLE
jupyter notebook gets "dead kernel" but jupyter console has no trouble

### ISSUE THREAD

**Issue by charukiewicz** [gibiansky/IHaskell#623]
I have ran into a bit of a problem with my IHaskell setup.  I am running IHaskell through a jupyter server on my VPS.  Here's what happened.

1. Last night I ran the `./ubuntu-install` script after cloning this repo
2. I configured and ran `jupyter notebook`.  Haskell notebooks were loading the kernel perfectly, I was able to connect to my server via my browser and use Haskell notebooks.
3. I powered off the server and resized the VPS from 1GB of RAM to 2GB of RAM.
4. Upon restarting, the jupyter notebook was getting "Dead Kernel" messages for Haskell notebooks.
5. I purged `~./ghc` and `~./cabal` and ran `./ubuntu-install.sh` again, it reinstalled again but did not work.
6. I purged the directories again and tried installing with `stack install` followed by `ihaskell install --stack`, and the notebook still does not work.

Here is the output of what I am getting now:

When running `jupyter notebook` and opening a Haskell notebook:
```
[W 14:50:18.784 NotebookApp] 404 GET /static/components/codemirror/mode/ihaskell/ihaskell.js?v=20151231143708 (xxx.xxx.xxx.xxx) 5.62ms referer=https://domain.com:4096/notebooks/Untitled.ipynb?kernel_name=haskell
ihaskell: Not in scope: ‘IHaskell.Eval.Widgets.relayWidgetMessages’
```

When running `ihaskell kernel`:
```
$ ihaskell kernel
No kernel profile JSON specified.
This may be a bug!
Args (Kernel Nothing) []
```

When running `jupyter console --kernel haskell` (this works properly):
```
$ jupyter console --kernel haskell
Jupyter Console 4.0.3

[ZMQTerminalIPythonApp] Loading IPython extension: storemagic

In [1]:
```

Does anyone know what the issue might be?  I feel that part of the problem might be that I did not remove the original install properly after the VPS resize, so there may be a conflict somewhere preventing a clean reinstall.

**TL;DR** - After resizing my VPS from 1GB to 2GB RAM, `jupyter notebook` will not launch the Haskell kernel but `jupyter console` will.

Thanks in advance.

**Comment by charukiewicz** [comment]
After some more playing with this I was able to solve the issue.  The problem was fixed simply by removing and re-cloning the local copy of the IHaskell repository and then rebuilding using the `ubuntu-install.sh` script.
